data/data/alibabacloud/variables-alibabacloud.tf

```diff
@@ -59,7 +58,6 @@ variable "ali_extra_tags" {
 
 Example: `{ "key" = "value", "foo" = "bar" }`
 EOF
-  default = {}
```

Comment:
Removing this default also requires removing the `omitempty` from the field in the tfvars struct.
https://github.com/openshift/installer/blob/c1c9b61f0c9b568af9d9ecd825a221e9f5b041f7/pkg/tfvars/alibabacloud/alibabacloud.go#L30